blockchain/node/src/chain_spec.rs

```diff
@@ -109,7 +116,11 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
                     get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
                     get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
                 ],
-                true,
+                FractalMintingConfig {
+                    fractal_authoritative_account: get_account_id_from_seed::<sr25519::Public>(
```

Comment:
I think the type for get_account_id_from_seed will be inferred. Feel free to keep for explicitness.